pydeseq2/DeseqDataSet.py

```diff
@@ -248,6 +256,8 @@ def fit_genewise_dispersions(self):
         Fits a negative binomial per gene, independently.
         """
 
+        p = self.design_matrix.shape[-1]
```

Comment:
I am never a fun of single letters as variable names therefore I would suggest to make it more meaningful

Reply:
Good idea, changed it to `num_vars` in [e536451](https://github.com/owkin/PyDESeq2/pull/32/commits/e5364518043f4edf9dccaf314ad8020bc7ba2b87)